test/index.html

```diff
@@ -183,6 +147,46 @@ <h1>Some Sample CFPB Charts</h1>
                      data-chart-type="bar">
                      This is the chart description.
                 </div>
+                <hr>
+                <h3>Indexed number of people with inquiries (auto)</h3>
```

Comment:
THANK YOU for adding headings so we can tell what the heck we're looking at in each chart! 😍